src/server.js

```diff
@@ -162,7 +162,7 @@ function initialiseGlobalMiddleware(app) {
     // The below adds the csrfToken to the res.render function which should hopefully
     // allow for local development without this hack
     const _render = res.render;
-    res.render = function addToken(view, options, fn) {
+    res.render = function (view, options, fn) {
```

Comment:
ESLint complains that it is anonymous, and SonarQube complains that it should be anonymous, so deferred to SonarQube in this instance.